apps/etterlatte-statistikk/src/main/resources/db/migration/V8__update_sak_view.sql

```diff
@@ -0,0 +1,3 @@
+CREATE OR REPLACE VIEW sak_statistikk AS
+    SELECT * FROM sak
+    WHERE tidspunkt_registrert > NOW() - interval '2 days';
```

Comment:
hvorfor de 2 seneste dagene?

Reply:
Det er litt magic inn i bildet, men det er en begrensning i BQ-spørringen i at den ikke kan dynamisk selecte det vi vil ha, og må ta inn en "fast" størrelse på spørring. Dette gjør den mot et view på tabellen som har med bare de to siste dagene.